app/frontend/src/submissions/components/SubmissionForm/ActivitySubmissionForm.vue

```diff
@@ -131,7 +137,6 @@ Licensed under the Apache License, Version 2.0 (the "License");
        :decommissionStartDate.sync="form.decommission_start_date"
        :decommissionEndDate.sync="form.decommission_end_date"
        :workStartDate.sync="form.work_start_date"
-       :workEndDate.sync="form.work_end_date"
```

Comment:
I'm removing this line as it's not referenced in the component.

I was running into an issue with an API test failing and removing this line has resolve that issue.